tests/conftest.py

```
@@ -73,8 +92,203 @@ def _override_get_db():
+        )
+        db_session.add(file_obj)
+        db_session.commit()
+        db_session.refresh(file_obj)
+        return file_obj
+    return _factory
+
+@pytest.fixture
+def remote_connection_factory(db_session: Session):
+    """Factory for RemoteConnection objects."""
+
+    def _factory(
```

Ruff (0.15.1)

[warning] Missing return type annotation for private function `_factory`

(ANN202)